python/tests/stress/arcticdb/version_store/test_mem_leaks.py

```diff
@@ -209,3 +383,201 @@ def proc_to_examine():
     max_mem_bytes = 608_662_528
 
     check_process_memory_leaks(proc_to_examine, 20, max_mem_bytes, 80.0)
+
+#endregion
+
+#region TESTS pytest-memray type for memory limit and leaks
+
+## NOTE: Currently tests can be executed on Python >= 3.8 only
+
+
+symbol = "test"
+
+
+@pytest.fixture
+def library_with_symbol(arctic_library_lmdb) -> Generator[Tuple[Library, pd.DataFrame], None, None]:
+    """
+        As memray instruments memory, we need to take out 
+        everything not relevant from mem leak measurement out of 
+        test, so it works as less as possible
+    """
```

Comment:
"""
    As memray instruments memory, we need to take out 
    everything not relevant from mem leak measurement out of 
    test, and place it in setup of the test - in other words in the fixture 
    Otherwise, memray instruments that code and it results in much slower execution
    as well as mixing results of memory leaks - are they in what we test - for example
    read() or in construct_df_querybuilder_tests() ? or in other code?
    """